lib/helpers/isAxiosError.js

```diff
@@ -7,5 +7,5 @@
  * @returns {boolean} True if the payload is an error thrown by Axios, otherwise false
  */
 module.exports = function isAxiosError(payload) {
-  return (typeof payload === 'object') && (payload.isAxiosError === true);
+  return (typeof payload === 'object') && (payload !== null) && (payload.isAxiosError === true);
```

Comment:
Can't we just check if `payload` is truthy?

```suggestion
  return !!payload && (typeof payload === 'object') && (payload.isAxiosError === true);
```